presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -5173,6 +5173,61 @@ public void testLateralJoin()
         assertQuery(
                 "SELECT * FROM (VALUES 1, 2) a(x) JOIN LATERAL(SELECT y FROM (VALUES 2, 3) b(y) WHERE y > x) c(z) ON z > 2*x",
                 "VALUES (1, 3)");
+
+        // Limit in correlated subquery
+        assertQuery(
```

Comment:
please add this test to `TestSubqueries` (and it would be nice to move also above non-connector related subquery tests as well in separate commit)

Reply:
Out of the three tests I added here, the one using tables cannot be moved to `TestSubqueries`. Shall I leave that one here and move the other two, or is there a way to test queries using tables in `TestSubqueries`?

Reply:
Yes, please.